compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticCode.java

```diff
@@ -420,6 +420,7 @@
     CANNOT_UPDATE_CONSTANT_VALUE("cannot.update.constant.value"),
     CANNOT_ASSIGN_VALUE_TO_CONSTANT("cannot.assign.value.to.constant"),
     CANNOT_DEFINE_CONSTANT_WITH_TYPE("cannot.define.constant.with.type"),
+    CANNOT_DEFINE_CONSTANT_WITH_IDENTIFIER("cannot.define.constant.with.identifier"),
```

Comment:
We already have a `UNDERSCORE_NOT_ALLOWED` error, how about re-using that instead? The error would be "underscore is not allowed here". @hasithaa 

If we need a separate error shall we change this to
```suggestion
    CANNOT_DEFINE_CONSTANT_WITH_UNDERSCORE("cannot.define.constant.with.underscore"),
```

Reply:
+1, better to use "underscore is not allowed here"

Reply:
Fixed. Used "Underscore not allowed" message